apps/maintainer-app/src/app/globals.css

```
@@ -45,60 +47,87 @@
+  }
 
   --animate-pulse: pulse var(--duration) ease-out infinite;
   @keyframes pulse {
-  0%, 100% {
-    boxShadow: 0 0 0 0 var(--pulse-color);}
-  50% {
-    boxShadow: 0 0 0 8px var(--pulse-color);}}
-  --animate-background-position-spin: background-position-spin 3000ms infinite alternate;
+    0%,
+    100% {
+      boxshadow: 0 0 0 0 var(--pulse-color);
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)